samples/esb/prx/sample.yaml

```diff
@@ -20,4 +20,15 @@ tests:
       - nrf5340dk_nrf5340_cpunet
     platform_allow: nrf52dk_nrf52832 nrf52833dk_nrf52833 nrf52840dk_nrf52840
       nrf52dk_nrf52810 nrf5340dk_nrf5340_cpunet
-    tags: ci_build
+    tags: esb ci_build
+  sample.esb.prx.dynamic_irq:
+    build_only: true
+    extra_configs:
+      - CONFIG_ESB_DYNAMIC_INTERRUPTS=y
+      - CONFIG_DYNAMIC_INTERRUPTS=y
```

Comment:
Isn't this option selected by `CONFIG_ESB_DYNAMIC_INTERRUPTS` in Kconfig?

Reply:
It is not. There is dependency to this option